src/lambda/lambdaService.ts

```diff
@@ -33,6 +33,59 @@ export class LambdaService {
       throw error
     }
   }
+
+  static invokeAsync ({
+    port = '',
+    region = 'sa-east-1',
+    functionName,
+    invocationType = 'RequestResponse',
```

Comment:
The lambda invoke has a specific type to invoke asynchronously. Based on my previous frustrating experiences, just dropping await doesn't make your invocation asynchronous, you need the correct type of invocation 😄
https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_RequestSyntax

Reply:
you're right